components/brave_wallet_ui/utils/api-utils.test.ts

```diff
@@ -1,19 +1,21 @@
 import { GetTokenParam, GetFlattenedAccountBalances } from './api-utils'
-import { mockNetworks } from '../stories/mock-data/mock-networks'
 import { AccountAssetOptions } from '../options/asset-options'
 import { mockAccount } from '../common/constants/mocks'
 
 describe('Check token param', () => {
   test('Value should return contract address', () => {
-    expect(GetTokenParam(mockNetworks[0], AccountAssetOptions[1])).toEqual('0x0D8775F648430679A709E98d2b0Cb6250d2887EF')
+    expect(GetTokenParam(AccountAssetOptions[1])).toEqual('0x0D8775F648430679A709E98d2b0Cb6250d2887EF')
```

Comment:
Anything you can do to extract token data to something we can reuse would be appreciated

Reply:
Done!